src/richie/apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -329,6 +329,37 @@ <h3 class="course-detail__title">
 
     {% block fragment_relations %}
     <div class="course-detail__relations">
+        {% block programs %}
+            {% with programs=course.get_programs %}
+                {% if programs %}
+                    {% autopaginate programs GLIMPSE_PAGINATION_PROGRAMS %}
+                    <div id="page{{ page_suffix }}" class="course-detail__programs course-detail__block">
+                        <div class="course-detail__row">
+                            <section class="course-glimpse-list">
+                                <h2 class="course-detail__title">
+                                    {% blocktrans count counter=programs|length %}
+                                        This course is part of a program
+                                    {% plural %}
+                                        This course is part of programs
+                                    {% endblocktrans %}
+                                </h2>
+                                <div class="section__items section__items--programs">
```

Comment:
When there is only one program in the `section__items`, the layout is broken. I suggested you a fix 235cb6da427bdcb246f93ab23371feee10aa6520

![Capture d’écran 2021-01-20 à 09 36 42](https://user-images.githubusercontent.com/9265241/105149702-20030e00-5b04-11eb-91f3-6c11ccd17775.png)
![Capture d’écran 2021-01-20 à 09 45 51](https://user-images.githubusercontent.com/9265241/105149868-517bd980-5b04-11eb-9fc9-6f94023e2306.png)